cmd/wallet-js-sdk/src/did/didmanager.js

```diff
@@ -92,10 +100,25 @@ export class DIDManager {
           keyType: DEFAULT_KEY_TYPE,
           update: true,
         },
+        {
+          id: keyAgreementKeySet.keyID,
+          type: keyAgreementType,
+          value: keyAgreementKeySet.publicKey,
+          encoding: "JWK",
+          keyType: keyAgreementKeyType,
+          purposes: ["keyAgreement"],
+        },
       ],
     };
 
-    let content = await this.agent.didclient.createOrbDID(createDIDRequest);
+    let content
+    try {
+      content = await this.agent.didclient.createOrbDID(createDIDRequest);
+    } catch (e) {
+      console.error('failed to create orbDID',e)
+      expect.fail(e);
```

Comment:
No need of `expect`, just throw JS error or let it throw error to caller (like before).

Reply:
`expect.fail()` statement removed